pallets/axelar-router/src/mock.rs

```diff
@@ -35,6 +35,8 @@ impl frame_system::Config for Runtime {
 impl cfg_mocks::router_message::pallet::Config for Runtime {
 	type Middleware = Middleware;
 	type Origin = DomainAddress;
+	type ReceiverMessage = Vec<u8>;
+	type SenderMessage = Vec<u8>;
```

Comment:
NIT. I think we're always using the same type for both, right? could we merge them under the same `Message` type. If, in the future, we need to use them with different types, the user can create different mock instances.

Reply:
[6a5cb5f](https://github.com/centrifuge/centrifuge-chain/pull/1971/commits/6a5cb5fdf6bee131476a92878f01183f3e5459e4)